src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/WriteTagCheckTest.java

```diff
@@ -52,144 +51,101 @@ protected String getPackageLocation() {
 
     @Test
     public void testDefaultSettings() throws Exception {
-        final DefaultConfiguration checkConfig = createModuleConfig(WriteTagCheck.class);
         final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
-        verify(checkConfig, getPath("InputWriteTagDefault.java"), expected);
+        verifyWithInlineConfigParser(getPath("InputWriteTagDefault.java"), expected);
     }
 
     @Test
     public void testTag() throws Exception {
-        final DefaultConfiguration checkConfig = createModuleConfig(WriteTagCheck.class);
-        checkConfig.addProperty("tag", "@author");
-        checkConfig.addProperty("tagFormat", "\\S");
         final String[] expected = {
-            "15: " + getCheckMessage(MSG_WRITE_TAG, "@author", "Daniel Grenner // violation"),
+            "15: " + getCheckMessage(MSG_WRITE_TAG, "@author", "must match pattern 'S'"),
```

Comment:
Here I had to change the message in the WriteTagCheckTest file because the tagFormat attribute in the resources file "InputWriteTag.java" was forcing me to write message in this way. (I have indicated that part in comment below).